src/main/java/ClientGet.java

```diff
@@ -0,0 +1,63 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClientGet {
+
+    public static void main(String[] args) throws IOException {
+
+
+        Currencies currenciesTable[] = Currencies.values();
+        List<String> currenciesExchangeRateStringList = new ArrayList<>();
+        List<CurrenciesExchangeRate> currenciesExchangeRateList = new ArrayList<>();
+
+
+        for (int j = 0; j < currenciesTable.length; j++) {
+            for (int i = 0; i < currenciesTable.length; i++) {
+                CurrenciesExchangeRate currenciesExchangeRate = new CurrenciesExchangeRate();
```

Comment:
Don't use setters- don't need them, just create constructor with 3 parameters and create this object later, just before you put in into your list.